packages/zoe/src/zoe.js

```diff
@@ -121,7 +149,8 @@ import { makeTables } from './state';
  * @property {Promise<OfferOutcome>} outcome Note that if the offerHook throws,
  * this outcome Promise will reject, but the rest of the OfferResultRecord is
  * still meaningful.
- * @property {(() => undefined)} [completeObj]
+ *
+ * @property {CompleteObj} [completeObj]
```

Comment:
what are the square brackets for?

Reply:
completeObj is only present in OfferResultRecord if exitKind was 'onDemand'. The square brackets mark it as optional.

Reply:
I think that's not what we want, in that the typescript will error if it is not included. If we actually want it to be optional, it should be:
```suggestion
 * @property {CompleteObj=} completeObj
```

The square brackets are ok, but I think we might fool ourselves into thinking that they are doing something with the tscheck

Reply:
done.